mods/cnc/maps/nod04a/map.yaml

```diff
@@ -0,0 +1,704 @@
+		Location: 45,11
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 40,17
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 39,34
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 21,38
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 20,28
+		Owner: Neutral
+	waypoint1: waypoint
+		Location: 13,26
+		Owner: Neutral
+	waypoint0: waypoint
+		Location: 13,14
+		Owner: Neutral
+	Civilian1: v33
+		Location: 48,34
+		Owner: Neutral
+		Health: 0.625
+	Civilian2: v33
+		Location: 54,37
+		Owner: Neutral
+	Soldier1: e2
+		Location: 22,15
+		Owner: GDI
+		Facing: 96
+		SubCell: 2
+	Soldier2: e1
+		Location: 17,14
+		Owner: GDI
+		Facing: 96
+		SubCell: 4
+	Soldier3: e2
+		Location: 13,16
+		Owner: GDI
+		Facing: 96
+		SubCell: 1
+	GDICiv1: c9
+		Location: 20,12
+		Owner: GDI
+		SubCell: 3
+	GDICiv2: c4
+		Location: 18,7
+		Owner: GDI
+		SubCell: 0
+	GDICiv3: c9
+		Location: 19,12
+		Owner: GDI
+		SubCell: 1
+	GDICiv4: c8
+		Location: 16,8
+		Owner: GDI
+		SubCell: 3
+	GDICiv5: c7
+		Location: 17,9
+		Owner: GDI
+		SubCell: 1
+	GDICiv6: c6
+		Location: 24,13
+		Owner: GDI
+		SubCell: 2
+	GDICiv7: c5
+		Location: 20,7
+		Owner: GDI
+		SubCell: 4
+	GDICiv8: c4
+		Location: 14,12
+		Owner: GDI
+		SubCell: 1
+	GDICiv9: c3
+		Location: 15,10
+		Owner: GDI
+		SubCell: 2
+	GDICiv10: c2
+		Location: 14,10
+		Owner: GDI
+		SubCell: 1
+	GDICiv11: c1
+		Location: 17,7
+		Owner: GDI
+		SubCell: 2
+	GDICiv12: c1
+		Location: 20,10
+		Owner: GDI
+		SubCell: 1
+	GDICiv13: c1
+		Location: 20,11
+		Owner: GDI
+		SubCell: 3
+	UnitsEntryBuggy: waypoint
+		Location: 52,6
+		Owner: Neutral
+	UnitsRallyBuggy: waypoint
+		Location: 52,10
+		Owner: Neutral
+	UnitsEntryBikes: waypoint
+		Location: 50,6
+		Owner: Neutral
+	UnitsRallyBikes: waypoint
+		Location: 50,10
+		Owner: Neutral
+	UnitsEntryGunner: waypoint
+		Location: 54,6
+		Owner: Neutral
+	UnitsRallyGunner: waypoint
+		Location: 54,10
+		Owner: Neutral
+	UnitsEntryRocket: waypoint
+		Location: 56,12
+		Owner: Neutral
+	UnitsRallyRocket: waypoint
+		Location: 54,12
+		Owner: Neutral
+	NodCiv1: c8
+		Location: 46,36
+		Owner: NodSupporter
+		Facing: 128
+		SubCell: 2
+	NodCiv2: e3
+		Location: 48,36
+		Owner: NodSupporter
+		Facing: 192
+		SubCell: 4
+	NodCiv3: c4
+		Location: 52,38
+		Owner: NodSupporter
+		Facing: 192
+		SubCell: 1
+	NodCiv4: c3
+		Location: 45,35
+		Owner: NodSupporter
+		Facing: 32
+		SubCell: 4
+	NodCiv5: c1
+		Location: 51,36
+		Owner: NodSupporter
+		Facing: 224
+		SubCell: 2
+	NodCiv6: c2
+		Location: 45,34
+		Owner: NodSupporter
+		Facing: 0
+		SubCell: 1
+	NodCiv7: c4
+		Location: 54,36
+		Owner: NodSupporter
+		Facing: 160
+		SubCell: 0
+	NodCiv8: c5
+		Location: 50,37
+		Owner: NodSupporter
+		Facing: 96
+		SubCell: 2
+	NodCiv9: c9
+		Location: 47,34
+		Owner: NodSupporter
+		Facing: 192
+		SubCell: 2	
+		
+Smudges:
+	sc6 37,24 0:
+	sc6 36,18 0:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+		EnemyWatcher:			
```

Comment:
Hm, yeah it is really not the rescan interval. I guess I'll send a patch myself, and clean some other things up as well.